pkgs/tools/security/fail2ban/default.nix

```diff
@@ -22,6 +22,8 @@ python3.pkgs.buildPythonApplication {
     })
   ];
 
+  propagatedBuildInputs = [ gamin ];
```

Comment:
Gamin does not support Python 3 and upstream does not recommend using it, hence the removal.

Reply:
Ok, removed.